From b595a7ba00fbe28583e37c89e95660e86186dc11 Mon Sep 17 00:00:00 2001
From: Oliver Brown <oliver.brown@freescale.com>
Date: Fri, 25 Jan 2013 13:22:27 -0600
Subject: [PATCH 229/249] ENGR00241363 Update VPU lib to accommodate FW
 v2.1.9, bug fix

Added CMD_SET_FRAME_DELAY to fix build for iMX53.

Change-Id: Ide8cd09a525a57e860641a377f6f28b9f0f5e6af
Signed-off-by: Oliver Brown <oliver.brown@freescale.com>
---
 vpu/vpu_reg.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/vpu/vpu_reg.h b/vpu/vpu_reg.h
index 2957d09..22ca28b 100644
--- a/vpu/vpu_reg.h
+++ b/vpu/vpu_reg.h
@@ -335,6 +335,7 @@
 #define CMD_SET_FRAME_DELAY		0x1BC
 #else
 #define CMD_SET_FRAME_MAX_DEC_SIZE      0x1A4
+#define CMD_SET_FRAME_DELAY		0x1BC
 #endif
 #define CMD_SET_FRAME_SOURCE_BUF_STRIDE 0x1A8
 /* End */
-- 
1.8.0

